src/components/Registration/StartDate.vue

```diff
@@ -54,53 +51,21 @@ export default class StartDate extends Mixins(DateMixin) {
 
   protected dateText = ''
 
-  /** The minimum start date that can be entered (up to 10 years ago today). */
-  get startDateMin (): Date {
-    const startDateMin = new Date(this.getCurrentJsDate) // make a copy
-    startDateMin.setFullYear(startDateMin.getFullYear() - 10)
-    startDateMin.setHours(0, 0, 0) // Set time to 0 for accurate Date Rules comparison
+  /** The minimum start date that can be entered (up to 10 years before today). */
+  get startDateMin (): string {
+    // no min date for staff
+    if (this.isRoleStaff) return null
 
-    return startDateMin
+    const date = new Date(this.getCurrentJsDate) // make a copy
+    date.setFullYear(date.getFullYear() - 10)
+    return this.dateToYyyyMmDd(date)
   }
 
-  /** The minimum start date string. */
-  get startDateMinStr (): string {
-    // no minimum start date restriction for staff
-    if (this.isRoleStaff) return ''
-
-    return this.dateToYyyyMmDd(this.startDateMin)
-  }
```

Comment:
I combined startDateMin() and startDateMinStr(). This is now almost the same as in Edit UI.

Ditto for max start date below.